iwyu.cc

```diff
@@ -1622,10 +1652,8 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
       return;
 
     // Figure out the best location to attribute uses inside macros.
-    if (IsInMacro(used_loc))
-      used_loc = GetUseLocationForMacroExpansion(used_loc, target_decl);
+    used_loc = GetCanonicalUseLocation(used_loc, target_decl);
```

Comment:
Comment is still correct but not as good as before. I don't know if you have better ideas to change the comment.